packages/python/patch-no-access

```diff
@@ -0,0 +1,14 @@
+--- Modules/posixmodule.c
++++ Modules/posixmodule.c
+@@ -2620,7 +2620,11 @@
+     }
+     else
+ #endif
++#ifdef HAVE_ACCESS
+         result = access(path->narrow, mode);
++#else
++        {result = -1; errno = EFAULT;}
```

Comment:
Could you maybe let this use `ENOSYS` instead of `EFAULT`? Also in some of the other patches. `EFAULT` may be misleading, because there is no actual memory fault occurring.